airbyte-integrations/connectors/source-bigquery/source_bigquery/streams.py

```diff
@@ -59,7 +65,16 @@ def next_page_token(self, response: requests.Response, **kwargs) -> Optional[Map
         return None
 
     def process_records(self, record) -> Iterable[Mapping[str, Any]]:
-        pass
+        fields = record.get("schema", {})["fields"]
+        stream_data = self.stream_data.read_records(sync_mode=SyncMode.full_refresh)
+
+        for data in stream_data:
+            rows = data.get("f")
+            yield {
+                "_bigquery_table_id": record.get("tableReference")["tableId"],
+                "_bigquery_created_time": record.get("creationTime"),
```

Comment:
This feels unintuitive that this is the `creationTime` of the table and not the record...

Reply:
I agree, I added a todo to update it later to row creation time

Reply:
Where is this todo so that I can track it?